### ISSUE TITLE
Update hotkeys when apps are reordered

### ISSUE THREAD

**Issue by cmeeren** [HenryNugraha/Window-List-Hotkey#1]
Currently the hotkeys aren't updated when apps are reordered. So if I open two apps and then change their order, Meta+1 will open the second window in the list, and vice versa.

**Comment by HenryNugraha** [comment]
Yes it is the current limitation as I have no way to know when and what apps are reordered.
Reordering by moving apps back and forth the workspace does the trick, but it's a bit of a hassle, I know. Please bear with it.

**Comment by cmeeren** [comment]
If it's not possible to know when the apps are reordered, is it possible to get the order of the apps anytime you want? If so, a possible solution might be to check the order for example every 10 seconds.

**Comment by HenryNugraha** [comment]
Just spent good 3 hours fidling with the code, unfortunately that is impossible too, sorry.
As far as I can tell, windows order is not being saved on any variable. I know it's a bit nonsense, but I can't find where the order list is being saved and I have no idea how to track them either.
All I have is a variable to store list of windows for each workspace, which will be updated when a window is added or removed. Also, since moving a window to another workspace involves adding and removing a window from certain workspace, the list is also updated.

**Comment by cmeeren** [comment]
Ok, thanks for trying. :+1:

**Comment by dandv** [comment]
Is it impossible to get the windows order due to [this bug](https://github.com/linuxmint/Cinnamon/issues/5516)?
